change-beta/@azure-communication-react-91789f9f-671a-47b8-b1f7-567a3546beee.json

```diff
@@ -0,0 +1,9 @@
+{
+  "type": "minor",
+  "area": "feature",
+  "workstream": "OnFetchProfile",
+  "comment": "move onfetchProfile to stable",
```

Comment:
Ideally this comment is the release note so the release driver doesn't have to go back to you and get that info